src/promptflow-core/promptflow/executor/_errors.py

```diff
@@ -82,6 +82,10 @@ def __init__(
         )
 
 
+class FlexFlowExecutionError(UserErrorException):
```

Comment:
ScriptExecutionError?

Reply:
I think Script is more clear than "FlexFlow"

Reply:
Refined.